docs/ReleaseHistory.md

```diff
@@ -11,6 +11,12 @@
 - FPS => False positive reduction in static analysis.
 - FNS => False negative reduction in static analysis.
 
+# UNRELEASED
+- BRK: Regular expression syntax has been standardized in JSON to conform to how the overwhelming majority of patterns were already defined.
+  - `refine` is used now used throughout as the name of the capture group used to isolate an actual find from the full expression that also matches delimiting characters. `secret` was previously used in some instances.
+  - `?<name>` is now used throughout for named captures. '?P<name>' was previously used in some instances. This may require replacing '?<' with '?P<' if using a regex engine that only accepts the '?P<name>' syntax.
+- BRK: `CachedDotNetRegexEngine`will no longer accept `(?P<name>)` syntax. This is only relevant if it is used with patterns other than those distributed with this library.
+- BUG: `SEC101/200.CommonAnnotatedSecurityKey` considered non-alphanumeric delimiter preceding secret to be part of the match.
```

Comment:
>CommonAnnotatedSecurityKey [](http://example.com/codeflow?start=19&length=26)

FWIW, this rule name is `UnclassifiedCommonAnnotatedSecurityKey` in other current project work.

Reply:
So I think this means I should leave it and let the change that renames it, rename it here too?